src/System.Security.SecureString/tests/SecureStringTests.cs

```diff
@@ -0,0 +1,250 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+using Xunit;
+
+public class SecureStringTest
+{
+    private static void VerifyString(SecureString ss, string exString)
+    {
+        IntPtr uniStr = SecureStringMarshal.SecureStringToCoTaskMemUnicode(ss);
+        string acString = Marshal.PtrToStringUni(uniStr);
+
+        Assert.Equal(exString.Length, acString.Length);
+        Assert.Equal(exString, acString);
+    }
+
+    private static SecureString CreateSecureString(string exValue)
+    {
+        SecureString ss = null;
+
+        if (string.IsNullOrEmpty(exValue))
+            ss = new SecureString();
+        else
+        {
+            unsafe
+            {
+                fixed (char* mychars = exValue.ToCharArray())
+                    ss = new SecureString(mychars, exValue.Length);
+            }
+        }
+
+        Assert.NotNull(ss);
+        return ss;
+    }
+
+    private static void CreateAndVerifySecureString(string exValue)
+    {
+        SecureString ss = CreateSecureString(exValue);
```

Comment:
SecureString is IDisposable.  Should we be explicitly disposing it when we're done?